scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py

```diff
@@ -137,16 +137,70 @@ def load_coord(coord_i):
                         )
             return coord, coordf
 
+        def load_ghost_coords(coord_i):
+            """
+            Here, ghost coordinates refers to coordinates that are offset from the physical coordinates
+            to include ghost zones. All coordinates are defined on vertices.
+
+            On completion:
+            coordg - ghost coords corresponding to the variable arrays
+            coordi - ghost coords of points on the interior of the block
+                     (which equals coordg when the data is only given in the interior)
+            coorde - ghost coords of the entire block
+                     (which equals coordg when the data includes ghost data)
+            coordi_ng - The non-ghost coordinates of the interior of the block
+            """
```

Comment:
I *think* understand what this means: you've grown each block to icnlude its ghost zones, and this requires "plotting" rather than physical coordinates, right?

Reply:
yes, not really sure what the best nomenclature to use is but essentially you need to add space in the coordinate system between blocks so the ghost zones don't overlap.

Reply:
Yeah I had trouble with this nomenclature, but I agree---I can't think of anything better.